.circleci/config.yml

```diff
@@ -106,14 +106,17 @@ jobs:
       - attach_workspace:
           at: ~/picassojs
       - run:
+          name: Run unit tests
           command: |
-            npx aw -c aw.config.js --mocha.bail false --mocha.reporter mocha-junit-reporter --mocha.reporterOptions.mochaFile ./coverage/junit/junit.xml
+            yarn run test:unit --maxWorkers=2 --coverage 
             if [ ! -z "$COVERALLS_REPO_TOKEN" ]; then
               echo "Uploading coverage results to coveralls.io..."
-              cat ./coverage/unit/lcov.info | npx coveralls
+              cat ./reports/lcov.info | npx coveralls
             fi
+          environment:
+            JEST_JUNIT_OUTPUT_DIR: ./reports/
       - store_test_results:
-          path: coverage/junit
+          path: ./reports/junit
```

Comment:
Im still not seeing anything under tests section in circleci?

Reply:
Working on it 😅